darts/utils/missing_values.py

```diff
@@ -64,7 +64,7 @@ def auto_fillna(ts: TimeSeries,
         A new TimeSeries with all missing values filled according to the rules above.
     """
 
-    ts_temp = ts.pd_series()
+    ts_temp = ts.pd_dataframe()
```

Comment:
Line 57: change `pandas.Series` to `pandas.DataFrame`.

Reply:
done